entry.sh

```diff
@@ -2,4 +2,4 @@
 
 echo '{"irods_host": "data.cyverse.org", "irods_port": 1247, "irods_zone_name": "iplant"}' | tee  > /home/user/.irods/irods_environment.json
 
-/usr/bin/tini -- ttyd zsh
\ No newline at end of file
+exec /usr/bin/tini -- ttyd tmux new -A -s ttyd zsh
```

Comment:
This means "if there's a tmux session called 'ttyd' already, connect to it; otherwise, start a new one with that name and run zsh" -- or, in other words, if the user loses their connection or closes the browser window, when they come back they won't have lost anything that was on their screen.

Reply:
Oh, and the `exec` means that the process gets replaced with the `tini` process, which is what needs to happen for tini to do things as it's supposed to (handling signals, etc.)